src/main/resources/assets/blocky-bubbles/lang/ru_ru.json

```
@@ -0,0 +1,13 @@
+{
+  "blocky-bubbles.options.culling_awareness.enum.bubble_column": "Пузырьковая колонка (подобная Java)",
+  "blocky-bubbles.options.culling_awareness.enum.non_air": "Невоздушные (паритет)",
+  "blocky-bubbles.options.culling_awareness.enum.none": "Без отбраковки",
+  "blocky-bubbles.options.culling_awareness.name": "Режим отбраковки",
+  "blocky-bubbles.options.culling_awareness.tooltip": "Какие блоки должны вызывать отбраковку соседних граней колонны пузырей.",
+  "blocky-bubbles.options.enable_animations.name": "Включить анимацию",
+  "blocky-bubbles.options.enable_animations.tooltip": "Следует ли анимировать текстуры колонны пузырей (требуется перезагрузка ресурсов).",
```

```suggestion
  "blocky-bubbles.options.enable_animations.tooltip": "Следует ли анимировать текстуры колонны пузырей (требует перезагрузки ресурсов).",
```
После применения изменений перезагрузка ресурсов происходит автоматически, поэтому лучше убрать возвратный суффикс, так как от игрока действий не требуется.